src/core/helpers/html/nl2br.ts

```diff
@@ -12,5 +12,5 @@
  *  Inserts HTML line breaks before all newlines in a string
  */
 export function nl2br(html: string): string {
-	return html.replace(/[\n\r]/g, '<br/>');
+	return html.replace(/\r\n|\r|\n/g, '<br/>');
```

Comment:
Need tests for this, and may be need \n\r too =)

Reply:
There are three different types of line breaks, all originally unique to the major operating systems: Windows/DOS, Macintosh, and Unix. A document using Mac line breaks would look horrid on a Windows system, and a document using Windows line breaks on Unix also wouldn’t be interpreted correctly. The cause for this is how the line break is actually created. The Mac, by default, uses a single carriage return (<CR>), represented as \r. Unix, on the other hand, uses a single linefeed (<LF>), \n. Windows goes one step further and uses both, creating a (<CRLF>) combination, \r\n.

https://www.oreilly.com/library/view/mac-os-x/0596004605/ch01s06.html#:~:text=There%20are%20three%20different%20types%20of%20line%20breaks%2C%20all,and%20uses%20both%2C%20creating%20a%20(%3CCRLF%3E)%20combination%2C%20%5Cr%5Cn.

Reply:
Can this test be adjusted for it: https://github.com/xdan/jodit/blob/a0b42f22a50b9d8aceb2d8e241c1fbdae14a1e3c/test/tests/acceptance/clipboardTest.js#L1199-L1219

Reply:
I suggest to already create a new jodit version which reverts the previous change or accepts this one to avoid putting the editor in a more broken state.